docs/current/sdk/nodejs/783645-get-started.md

```diff
@@ -0,0 +1,195 @@
+- It uses the client's `container().from()` method to initialize a new container from a base image. This base image is the Node.js version to be tested against - the `node:16` image. This method returns a new `Container` object with the results.
+- It uses the `Container.withMountedDirectory()` method to mount the host directory into the container at the `/src` mount point, and the `Container.withWorkdir()` method to set the working directory in the container. The revised `Container` is stored in the `runner` constant.
+- It uses the `Container.exec()` method to define the command to run tests in the container - in this case, the command `npm test -- --watchAll=false`.
+- It uses the `Container.exitCode()` method to execute the command and obtain the corresponding exit code. An exit code of `0` implies successful execution (all tests pass).
+- It invokes the `Container.exec()` method again, this time to define the build command `npm run build` in the container.
+- It obtains a reference to the `build/` directory in the container with the `Container.directory()` method. This method returns a `Directory` object.
+- It writes the `build/` directory from the container to the host using the `Directory.export()` method.
+
+:::tip
+The `from()`, `withMountedDirectory()`, `withWorkdir()` and `exec()` methods all return a `Container`, making it easy to chain method calls together and create a pipeline that is intuitive to understand.
+:::
+
+Run the Node.js CI tool by executing the command below:
+
+```shell
+node --loader ts-node/esm ./build.ts
+```
+
+The tool tests and builds the application, logging the output of the test and build operations to the console as it works. At the end of the process, the built React application is available in a new `build` folder in the project directory, as shown below:
+
+```shell
+tree build
+build
+├── asset-manifest.json
+├── favicon.ico
+├── index.html
+├── logo192.png
+├── logo512.png
+├── manifest.json
+├── robots.txt
+└── static
+    ├── css
+    │   ├── main.073c9b0a.css
+    │   └── main.073c9b0a.css.map
+    ├── js
+    │   ├── 787.28cb0dcd.chunk.js
+    │   ├── 787.28cb0dcd.chunk.js.map
+    │   ├── main.f5e707f0.js
+    │   ├── main.f5e707f0.js.LICENSE.txt
+    │   └── main.f5e707f0.js.map
+    └── media
+        └── logo.6ce24c58023cc2f8fd88fe9d219db6c6.svg
+```
+
+## Step 4: Test against multiple Node.js versions
+
+Now that the Node.js CI tool can test the application against a single Node.js version, the next step is to extend it for multiple Node.js versions.
+
+Replace the `build.ts` file from the previous step with the version below (highlighted lines indicate changes):
+
+```typescript file=snippets/get-started/step4/build.ts
+```
+
+This revision of the CI tool does much the same as before, except that it now supports multiple Node.js versions.
```

Comment:
This sentence seems off to me. Maybe change to "This revision of the CI tool is the same as before, except that it now supports multiple Node.js versions.

Reply:
Modified